imports/plugins/included/marketplace/lib/collections/schemas/marketplace.js

```diff
@@ -64,10 +64,10 @@ export const MarketplacePackageConfig = PackageConfig.clone().extend({
     type: Array,
     defaultValue: [{
       shopType: "merchant",
-      active: true
+      enabled: true
```

Comment:
[see the Schema definition above](https://github.com/reactioncommerce/reaction/pull/4334/files#diff-18f984780eed6839f9ac58fabffbe9dcR19)